src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.scss

```diff
@@ -2,44 +2,30 @@
 @import 'src/content/app/entity-viewer/gene-view/styles/constants';
 
 .container {
-  display: grid;
-  grid-template-columns: $left_column $middle_column_left_gap auto;
-  padding: 9px 0;
-}
-
-.filterLabel {
-  grid-column: 1/2;
-  text-align: right;
-  padding-right: 2px;
-  cursor: pointer;
-}
-
-.chevron {
-  margin-left: 10px;
-  height: 12px;
-  width: 12px;
-  margin-bottom: -3px;
+  grid-area: 3/1/3/4;  
+  position: sticky;
+  height: 100%;
+  top: -5px;
+  z-index: 1;  
```

Comment:
Do we need these two properties? Like `top` above, I don't see any difference after removing them.

Reply:
Thats again to keep it at the right position when it is fixed when scrolling the page.